### ISSUE TITLE
LogSoftMax conversion

### ISSUE THREAD

**Issue by thunder3057** [AlexanderLutsenko/nobuco#15]
Hi,

I am getting the following error due to the use of `torch.nn.functional.log_softmax`.
So I wrote the following conversion for it:
```
@nobuco.converter(torch.nn.functional.log_softmax, channel_ordering_strategy=nobuco.ChannelOrderingStrategy.MINIMUM_TRANSPOSITIONS)
def log_softmax(input: torch.Tensor, dim: int = -1):
    return lambda input, dim=dim: tf.nn.log_softmax(input, axis=dim)
```

But I am still getting an error from the nobuco converter:
`Exception: Failed conversion: <function log_softmax at 0x7fa24fdf9820>`

This is the class where it fails at:
```
class OutBlock(nn.Module):
    def __init__(self):
        super(OutBlock, self).__init__()
        self.conv = nn.Conv2d(256, 1, kernel_size=1)  # value head
        self.bn = nn.BatchNorm2d(1)
        self.fc1 = nn.Linear(10 * 10, 128)
        self.fc2 = nn.Linear(128, 1)

        self.conv1 = nn.Conv2d(256, 128, kernel_size=1)  # policy head
        self.bn1 = nn.BatchNorm2d(128)
        self.logsoftmax = nn.LogSoftmax(dim=1)
        self.fc = nn.Linear(10 * 10 * 128, 10 * 10 * 36)

    def forward(self, s):
        v = F.relu(self.bn(self.conv(s)))  # value head
        v = v.view(-1, 10 * 10)  # batch_size X channel X height X width
        v = F.relu(self.fc1(v))
        v = torch.tanh(self.fc2(v))

        p = F.relu(self.bn1(self.conv1(s)))  # policy head
        p = p.view(-1, 10 * 10 * 128)
        p = self.fc(p)
        p = self.logsoftmax(p).exp()
        return p, v
```
 
 
 Thanks in advance!

**Comment by AlexanderLutsenko** [comment]
@thunder3057 Hey! In the console output, you can find this warning:
```console
UserWarning: Conversion exception on node 'log_softmax': log_softmax() got an unexpected keyword argument '_stacklevel'
```
`torch.nn.functional.log_softmax` does indeed require this weird `_stacklevel` arg. 
I'd advise to provide a converter for `torch.softmax` and `torch.Tensor.softmax` instead as those are the most low-level operations out there. 
Also, you chose `MINIMUM_TRANSPOSITIONS` strategy, it's efficient, but that means the converted node may receive the input tensor in both channel-first (pytorch) OR channel-last (tensorflow) order. If the order is channel-last, which you can check for, you must change `dim` accordingly. 

To sum it up:

```python
@converter(torch.log_softmax, torch.Tensor.log_softmax, channel_ordering_strategy=ChannelOrderingStrategy.MINIMUM_TRANSPOSITIONS)
def converter_log_softmax(input: Tensor, dim, *, dtype: Optional[_dtype]=None):
    num_dims = input.dim()

    def func(input: Tensor, dim, *, dtype=None):
        if get_channel_order(input) == ChannelOrder.TENSORFLOW:
            dim = dim_pytorch2keras(dim, num_dims)
        return tf.nn.log_softmax(input, axis=dim)
    return func
```
I just added this in `v0.7.1`, should work out of the box now.
